static/docs/command-reference/remote/add.md

```diff
@@ -234,6 +234,35 @@ $ dvc remote add myremote "azure://"
 
 <details>
 
+### Click for Google Drive
+
+> Since Google Drive has tight API usage quotas, creation and configuration of
```

Comment:
let's not do this as a Note (with `>`) - it's an essential part of the workflow.

Please, let's put some comments about auth flow and the credentials file. Probably after the `dvc remote add ...` commands.